<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="DetailView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 191, 191);</string>
    </property>
    <widget class="QWidget" name="DetailViewtContents"/>
   </widget>
   <widget class="QPlainTextEdit" name="OutputView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QDockWidget" name="ListView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>243</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 191, 191);</string>
    </property>
    <widget class="QWidget" name="ListViewWidgetContents"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
